packages/docusaurus-theme-translations/locales/vi/theme-common.json

```diff
@@ -61,10 +61,10 @@
   "theme.docs.sidebar.expandButtonTitle": "Mở rộng thanh bên",
   "theme.docs.sidebar.navAriaLabel": "Thanh điều hướng tài liệu",
   "theme.docs.sidebar.toggleSidebarButtonAriaLabel": "Đóng - mở thanh điều hướng",
-  "theme.docs.tagDocListPageTitle": "{nDocsTagged} với \"{tagName}\"",
+  "theme.docs.tagDocListPageTitle": "{nDocsTagged} được gắn thẻ \"{tagName}\"",
   "theme.docs.tagDocListPageTitle.nDocsTagged": "{count} tài liệu đã gắn thẻ",
   "theme.docs.versionBadge.label": "Phiên bản: {versionLabel}",
-  "theme.docs.versions.latestVersionLinkLabel": "phiên bản mới nhất",
+  "theme.docs.versions.latestVersionLinkLabel": "Phiên bản mới nhất",
```

Comment:
Similarly I don't think we want uppercase here. The base language doesn't.

The final sentence would be:

> Để xem các cập nhật mới nhất, vui lòng xem phiên bản **Phiên** bản mới nhất


Code: 

```js
function LatestVersionSuggestionLabel({
  versionLabel,
  to,
  onClick,
}: {
  to: string;
  onClick: () => void;
  versionLabel: string;
}) {
  return (
    <Translate
      id="theme.docs.versions.latestVersionSuggestionLabel"
      description="The label used to tell the user to check the latest version"
      values={{
        versionLabel,
        latestVersionLink: (
          <b>
            <Link to={to} onClick={onClick}>
              <Translate
                id="theme.docs.versions.latestVersionLinkLabel"
                description="The label used for the latest version suggestion link label">
                latest version
              </Translate>
            </Link>
          </b>
        ),
      }}>
      {
        'For up-to-date documentation, see the {latestVersionLink} ({versionLabel}).'
      }
    </Translate>
  );
}
```